tarodb/asset_minting.go

```diff
@@ -277,6 +279,19 @@ func (a *AssetMintingStore) CommitMintingBatch(ctx context.Context,
 
 		return nil
 	})
+
+	sqlError := ParseSqlError(err)
```

Comment:
I think we should use `ParseSqlError` in `ExecTx` to always do the transformation. And maybe call it `MapSQLError`?